### PR TITLE
ZEPPELIN-3137. Improve code style check

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Although currently zeppelin has code check style, it is very weak. Such as unused import is not detected. The is the first PR for adding code style check. It just improve the code style check file , but just disable it for now. I have created subtask under ZEPPELIN-3137 for each module.  This `checkstyle.xml` is from spark project. I think code style of spark is also fit for zeppelin project, just remove some spark related content in this `checkstyle.xml`

### What type of PR is it?
[ Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3137

### How should this be tested?
* Travis build pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
I didn't check all the rules one by one. Just copy the checkstyle from spark project and remove some spark specific thing. Things like Indentation & UnusedImports is changed.

**Comment by zjffdu** [comment]
Just update the PR to make diff more readable. @felixcheung

File: _tools/checkstyle.xml [hunk 1]
```diff
@@ -279,5 +279,14 @@ limitations under the License.
...
     </module>
+
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="expected"/>
+    </module>
+    <module name="CommentsIndentation"/>
+    <module name="UnusedImports"/>
+    <module name="RedundantImport"/>
+    <module name="RedundantModifier"/>
+    <module name="AvoidStarImport"/>
```
Comment (felixcheung) [inline _tools/checkstyle.xml]: ok I'd agree with these...

**Comment by zjffdu** [comment]
Hmm, make sense. Enable the import order checking

File: _tools/checkstyle.xml [hunk 2]
```diff
@@ -58,10 +58,11 @@ limitations under the License.
       <!-- Checks for redundant import statements. -->
       <property name="severity" value="error"/>
     </module>
-    <!-- <module name="ImportOrder"> Checks for out of order import statements
+    <module name="ImportOrder"> Checks for out of order import statements
       <property name="severity" value="warning"/> <property name="groups" value="com.google,android,junit,net,org,java,javax"/>
```
Comment (felixcheung) [inline _tools/checkstyle.xml]: I'd add `scala` then `org.apache` there?
perhaps separate `org.apache.zeppelin` too?

(maybe remove `andriod`?)

Comment (zjffdu) [reply _tools/checkstyle.xml]: Fixed

**Comment by jongyoul** [comment]
LGTM

**Pull request closed without merging**